assets/instance/userdata.sh

```diff
@@ -60,10 +60,16 @@ export -f provision_instance_stores
 
 # --- install: core -----------------------------
 
+
+
 yum upgrade -y
 
 dnf install -y docker
 
+dnf upgrade --refresh rpm glibc
+rm /var/lib/rpm/.rpm.lock
+dnf -y update
+
```

Comment:
How does this related to adding a key pair? Maybe it should be removed? If it is required, I provided a suggestion to remove the extra white spaces.

```suggestion
# --- install: core -----------------------------

yum upgrade -y

dnf install -y docker

dnf upgrade --refresh rpm glibc
rm /var/lib/rpm/.rpm.lock
dnf -y update
```